packages/eas-cli/src/commands/update/list.ts

```diff
@@ -0,0 +1,102 @@
+import { getConfig } from '@expo/config';
+import { Command, flags } from '@oclif/command';
+import Table from 'cli-table3';
+
+import { getUpdates } from '../../../src/update/updateUtils';
+import log from '../../log';
+import { ensureProjectExistsAsync } from '../../project/ensureProjectExists';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+
+export default class UpdateList extends Command {
+  static hidden = true;
+  static description = 'View a list of updates.';
+
+  static args = [
+    {
+      name: 'releaseName',
+      required: false,
+      description: 'Name of the release to view',
+    },
+  ];
+
+  static flags = {
+    platform: flags.string({
+      description: 'Platform-specifc updates to return: ios, android, web.',
+    }),
+    all: flags.boolean({
+      description: 'Return a list of all updates individually.',
+      default: false,
+    }),
+    json: flags.boolean({
+      description: 'Return list of updates as JSON.',
+      default: false,
+    }),
+  };
+
+  async run() {
+    let {
+      args: { releaseName },
+      flags: { json: jsonFlag, platform: platformFlag, all: allFlag },
+    } = this.parse(UpdateList);
+
+    const projectDir = await findProjectRootAsync(process.cwd());
+    if (!projectDir) {
+      throw new Error('Please run this command inside a project directory.');
+    }
```

Comment:
I see this flow a lot, I'm wondering if I should make a PR to just move this into `findProjectRootAsync` @fson do you have an opinion?